src/eql.erl

```diff
@@ -9,6 +11,26 @@ compile(File) ->
         Error        -> Error
     end.
 
+compile(Tab, File) ->
+    case compile(File) of
+        {ok, Queries} ->
+            [ets:insert(Tab, Query) || Query <- Queries],
```

Comment:
Woops! Thanks, it was originally using `set_env` so needed the name separate :)

I've changed it, squashed and pushed.